.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules
 .DS_Store
 yarn-error.log
 lib
-generated
\ No newline at end of file
+generated
+.env
+*.db
```

Comment:
did you want the db-journal to be committed?

Reply:
Nop, added to the ignore